### DISCUSSION TITLE
ZUGFeRD plugin not working properly

### DISCUSSION THREAD

**Discussion by 112marv** [pretix/pretix#4724] [Q&A] [answered]
Hi,
I have installed a self-hosted Pretix instance on a server via Docker. I have created a new Docker image with Ghostscript and the Pretix-ZUGFeRD plugin as described in the [documentation](https://docs.pretix.eu/en/latest/admin/installation/docker_smallscale.html#install-a-plugin). I also edited the pretix.cfg and added the location of ghostscript. The plugin is displayed under the plugins, I can activate it and it is displayed in the settings.
However, not like this screenshot [here](https://pretix.eu/about/de/blog/20241218-e-rechnung-starten/#:~:text=Danach%20k%C3%B6nnen%20Sie,zur%20entsprechenden%20Option%3A), but only simplified (see screenshot). I have also selected an Invoice Style with ZUGFeRD profile XRECHNUNG in the invoice settings. The created invoice in pdf format unfortunately has no xml file as an attachment. What could I have done wrong? Does anyone have any tips for me?

```
FROM pretix/standalone:2024.11.0
USER root
RUN apt update && apt install ghostscript -y
RUN pip3 install pretix-zugferd
USER pretixuser
RUN cd /pretix/src && make production
```

![settings](https://github.com/user-attachments/assets/a46ff6ce-ee8c-4c12-8de3-f5a0aa9f9dd0)

**Comment by raphaelm** [comment, marked as answer]
The screenshot from the blogpost is from the current development version of the plugin, it's not yet released as a stable version. This will happen at the end of the month.

**Comment by 112marv** [comment]
It works with todays update. Thx a lot.
